data/json/items/vehicle/engine.json

```diff
@@ -344,6 +344,8 @@
     "name": { "str": "25 L 650 HP gas turbine engine" },
     "description": "A gas turbine engine.  Used primarily for small helicopters.",
     "//": "Based on Allison 250-C30R.  Doubled the published weight to account for a transmission.",
+    "//2": "It's from 60s, tempered steel might be appropriate?"
+    "material": [ "superalloy" ],
```

Comment:
Please refrain from using superalloy in mundane items, ideally it should be removed, and left only for mods like aftershock

Reply:
So superalloy is sci-fi material from far future? Because some of the materials used for modern gas turbines / jet engines are literally named superalloys: see https://cdn.intechopen.com/pdfs/22905/InTech-Materials_for_gas_turbines_an_overview.pdf

I couldn't find any info on specific steel grades / alloys used in these engines, but using a tempered steel (that can just be made by in a small workshop) didn't feel right. Is there a high-end steel in the game, or is tempered steel a decent proxy for the game?

Reply:
The Allison 250-C30R is a little different: given its age, I suspect the original materials are close to modern high-speed steels (strong, temperature-resistant) so I believe it might be a good fit for the game

Reply:
Yeah, even if the name is the same, in dda superalloy was used a lot as catchall for random super material type used in whatever author wanted, including superalloy vehicle rams and superalloy dog harnesses. Which, of coursed, poisoned the term similarly to "survivor" line of stuff, so moving away from it is "generally good". You can, of course, add new "high-performance alloy" material and items to represent this thing better, but i don't think we really need to go this deep in our abstractions

Reply:
I really don't want to add a new material type, that's way out of scope of what I am trying to do. (I just want to edit existing things so they would stop using `"steel"` material, and fixing obvious mistakes along the way.)

I converted the motors to tempered steel. The only superalloy vehicle stuff remaining is superalloy plating, and it works as is, so I am not touching it.